src/app/commons/components/charts/charts.component.ts

```diff
@@ -8,13 +8,20 @@ export class ChartsComponent implements OnInit {
   data: any;
     
   options: any;
+  
+  type: String;
```

Comment:
lets remove this type as mentioned in the previous comments

Reply:
This is the property of chart component, this I can't remove but as you mentioned above I removed `type` property from bugs-technology an bugs-dashboard component.

Reply:
agreed